src/oods.cairo

```diff
@@ -0,0 +1,34 @@
+use cairo_verifier::air::composition::eval_composition_polynomial;
+use cairo_verifier::air::global_values::InteractionElements;
+use cairo_verifier::air::public_input::PublicInput;
+
+struct OodsValues {
+    mask_values: Array<felt252>,
+    split_polynomials: Array<felt252>
```

Comment:
Given this has 2 values, let's make it a tuple or dedicated struct
Array is overkill i think